### DISCUSSION TITLE
looking for easy way to stop \, becoming ,

### DISCUSSION THREAD

**Discussion by sh9temp** [jgm/pandoc#9916] [Q&A] [answered]
not serious
While converting markdown to latex,
outside `$...$`, `\,` becomes `,`.
`\;\!` has the same problem.
They can be used at text mode in LaTeX.
Now escape has no meaning because every symbol can be input without `\`.
`\<\>` has similar problem, but `<>` is used for tags so I can understand.
`\\\>` cannot be `\>` because `\\` must be `\textbachslash` and `\>` must be `\textgreater`.
I know to use `` `...`{=latex}``, but isn't there any way easier?
always doing like
```markdown
100`\,`{=latex}\textcelsius.
```
is stressful.
I want to use markdown syntax so making paragraph raw content is not useful.
Is there any good option or extension or anything else?
now I'm using a small script to substitute them to raw attribute.
not serious

**Comment by jgm** [comment, marked as answer]
This is the standard way to escape these symbols in markdown.  In markdown, `\,` means exactly the same as `,`.

If you're trying to include raw LaTeX to be passed through to LaTeX output, then using the raw attribute as you do is the way to go.

If that's unergonomic, you could consider using a Lua filter to make what you're doing easier.  Or, e.g., if you always want `\,` before `\textcelsius`, then define a new macro `\mytextcelsius` that does both and use this.

**Comment by iandol** [comment]
Switching to unicode like 100° (U+00B0) or 100℃ (U+2103) may also reduce the need for escaping somewhat...

**Comment by sh9temp** [comment]
Oh I forgot that I can define latex macro in pandoc! Pandoc realizes that it is latex code and never remove the escape. And yes pandoc automatically includes unicode-math so I can use unicode characters and it makes the source document readable.  All problem was solved. Thank you very much!
